static/js/components/HeaderContent.jsx

```diff
@@ -1,21 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
+import Box from '@material-ui/core/Box';
 import Responsive from "./Responsive";
 import ProfileDropdown from "./ProfileDropdown";
 import Logo from "./Logo";
+import QuickSearchBar from "./QuickSearchBar";
 
 import styles from "./Main.css";
 
 const HeaderContent = () => (
   <div className={styles.topBannerContent}>
-    <div style={{ display: "inline-block", float: "left" }}>
+    <div style={{ display: "inline-flex", flexDirection: "row" }}>
       <Logo className={styles.logo} />
       <Link className={styles.title} to="/">
-        SkyPortal ∝
+        SkyPortal
       </Link>
+      <Box p={2} />
+      <QuickSearchBar id="search" />
     </div>
-    <div style={{ position: "fixed", right: "1em" }}>
+    <div style={{ position: "fixed", right: "1em", top: "1em" }}>
```

Comment:
This one too.

Reply:
Would you like me to fix the right: 1em as well?

Reply:
Already fixed in closed PR #673